package.json

```
@@ -30,7 +30,7 @@
         "@babel/env",
         {
           "targets": {
-            "node": "10"
+            "node": "12"
```

This change seemed to have no effect on the generated snapshot sizes, wondering if this config can simply be removed?

---

Removing this config seems to need another plugin, as I proposed in PR #11.

Increasing this "target" value to `16` does not seem to affect the results of `npm test` or `yarn test-build`.

I may split this update into a separate proposal.

---

reverting the change here, moving out to PR #12.

```suggestion
            "node": "10"
```